src/app/Processor/ProcessorEdit/ProcessorEdit.tsx

```diff
@@ -431,7 +431,11 @@ const ProcessorEdit = (props: ProcessorEditProps): JSX.Element => {
                                 </p>
                               )}
                             </FormSection>
-                            <FormSection title={t("processor.action")}>
+                            <FormSection
+                              title={t("processor.action")}
+                              data-ouia-component-id="actions"
+                              data-ouia-component-type="SE/FormSection"
```

Comment:
similar question as above

Reply:
@jomarko same situation it is named by the RHOSE project -> SE as SmartEvent project